### PR TITLE
Fix button disable state and reactivity

### LINKED ISSUE #2621: Disabled primary button unreadable
image

cc @marcoambrosini

### PR DESCRIPTION
**Pull request by skjnldsv**
- [x] Update documentation
- [x] Fix reactivity
- [x] Fix disabled design

Peek 08-04-2022 10-07

Fix #2621 

### Dark theme + different primary colour
Peek 08-04-2022 10-15

### Dark theme + black primary colour
Peek 08-04-2022 10-16

### Light theme + black primary colour
Peek 08-04-2022 10-18

### Light theme + bright primary colour
Peek 08-04-2022 10-19

### CODE REVIEW

**Review by marcoambrosini — APPROVED** [review]
amazing :)

**Comment by raimund-schluessler** [comment]
Very nice! I just wonder whether we should disable the hover feedback for disabled buttons. They still look quite enabled when you hover them. :thinking:

File: src/components/Button/Button.vue [hunk 1]
```diff
@@ -327,21 +292,22 @@ export default {
...
 			 */
 			tabbed: false,
+
 			/**
-			 * $slots are not reactive.
-			 * We need to update the content manually
+			 * Making sure the slots are reactive
 			 */
-			text: this.getText(),
+			slots: this.$slots
```
Comment (raimund-schluessler) [inline src/components/Button/Button.vue:299]: Lint issues.

**Comment by skjnldsv** [comment]
Added more examples

**Comment by skjnldsv** [comment]
> Very nice! I just wonder whether we should disable the hover feedback for disabled buttons. They still look quite enabled when you hover them. :thinking:

Done :rocket:

**Review by raimund-schluessler — APPROVED** [review]
Great!

**Pull request merged**